src/servers/ZoneServer2016/entities/constructiondoor.ts

```diff
@@ -73,7 +94,7 @@ export class ConstructionDoor extends DoorEntity {
     this.slot = slot;
     this.profileId = 999; /// mark as construction
```

Comment:
custom profileIds should be a constant

Reply:
i know it's not from that pr, it's just a reminder so i will create an issue for it